### DISCUSSION TITLE
启动时固定打开某个wiki或某些wiki

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TidGi-Desktop#579] [Ideas]
好像还没有这个功能。现在我打开太记，就会默认启动上次最后使用的wiki，但有些时候，某些wiki我也想立即加载出来。

**Comment by linonetwo** [comment]
如果没设置为休眠的话，应该就是在后台启动了，可以随时用「在新窗口打开」来同时开启

**Reply by dongrentianyu** [reply]
对，但有些时候就需要重新加载等一等。所以想有个功能专门固定下来。确定某些wiki是开机启动时一定会加载的。即使上次没打开也一定会加载出来。

**Reply by linonetwo** [reply]
默认就是这样的，不开休眠的话，所有wiki都是同时启动的。你可以新建一个工作区看看，重启后等一会儿打开它，应该是秒开，因为已经启动了。

**Reply by dongrentianyu** [reply]
![image](https://github.com/user-attachments/assets/f8df583b-fa95-40bc-9e5b-6a6d206b9e86)

哦对，我想起来了。之前我就是觉得太记占用内存比较高，就开启了休眠的功能。但开了吧，一些不经常使用的wiki感觉也没必要加载，毕竟会影响性能嘛。但只启动最后一个使用的wiki，那我经常想打开的那个wiki就还需要再启动一遍。

即经常使用或者想要固定打开的wiki多数时候是最后一个使用的wiki，但有时候不是，所以还得启动一下。

**Reply by linonetwo** [reply]
可以每个单个wiki设置这个

**Reply by dongrentianyu** [reply]
也就是反过来，在主设置里关闭上面的休眠所有工作区，然后在每个单个wiki里设置开启休眠。懂了，这样确实也可以。
